can_tools/bootstrap_data/covid_demographics.csv

```diff
@@ -387,4 +387,10 @@ all,asian_or_pacific_islander,non-hispanic,all
 55-64,white,all,male
 55-64,white,all,female
 65_plus,white,all,male
-65_plus,white,all,female
\ No newline at end of file
+65_plus,white,all,female
+5-11,ai_an,all,all
```

Comment:
Not sure if we want to zero-pad these values or not, but most dashboards seem to return values without padding (e.g "5-11" instead of "05-11") so I stuck with that. I think it is important that we keep this standardized either way

Reply:
👍  no zero-padding sounds good to me.